recipes/lazy_loader/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "lazy_loader" %}
+{% set version = "0.1rc2" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/lazy_loader-{{ version }}.tar.gz
+  sha256: 1130fe8ddf64e7bfbeea185e14a08575b55a4680ed1585482eaeb796faca242b
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - flit-core >=3.3,<4
+    - python >=3.8
+  run:
+    - python >=3.8
+
+test:
+  imports:
+    - lazy_loader
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://pypi.org/project/lazy_loader/
+  summary: lazy_loader
```

Comment:
Can you add some more info here?

Reply:
Ok. Will do. 

💡 Didn't realize (pay attention) the summary provided by `grayskull` was so over summarized!

Reply:
Done.